build.sbt

```diff
@@ -63,6 +63,7 @@ lazy val tests = crossProject
     hasITs,
     coursierPrefix,
     libs += Deps.scalaAsync.value,
+    pureJava,
```

Comment:
That shouldn't be necessary, it's only for Java-only projects. The scala-js errors can be addressed…